cmd/podman/common/create.go

```diff
@@ -544,6 +544,18 @@ func DefineCreateFlags(cmd *cobra.Command, cf *ContainerCLIOpts) {
 	)
 	_ = cmd.RegisterFlagCompletionFunc(podIDFileFlagName, completion.AutocompleteDefault)
 
+	// Defining brand-new flag for TLS verification, so that `run` and `create` commands can make use of it.
```

Comment:
```suggestion
	// Flag for TLS verification, so that `run` and `create` commands can make use of it.
```
Once merged, it's not brand new anymore :)

Reply:
Haha, Good One! :-D

Done changed!